llvm/unittests/Analysis/VectorFunctionABITest.cpp

```diff
@@ -423,108 +447,167 @@ TEST_F(VFABIParserTest, MissingVectorNameTermination) {
 }
```

Comment:
definitely not for this patch, but at some point would be good to group somehow those tests in this file :)